src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js

```diff
@@ -31,7 +35,43 @@ export default class geTemplateBuilderFormField extends LightningElement {
     @wire(getObjectInfo, { objectApiName: '$targetObjectApiName' })
     wiredObjectInfo(response) {
         if (response.data) {
-            this.objectDescribeInfo = response.data;
+            this.targetObjectDescribeInfo = response.data;
+            this.validate();
+        }
+    }
+
+    renderedCallback() {
+        if (!this.hasRendered) {
+            this.hasRendered = true;
+
+            if (this.objectApiName === DI_BATCH_INFO.objectApiName) {
+                return;
+            }
+
+            if (isEmpty(this.fieldMapping)) {
+                const inputField = this.template.querySelector('[data-id="formField"]');
+
+                inputField.setCustomValidity('Field not found');
```

Comment:
Missing custom label for this string.

Reply:
this was resolved in a later commit. thanks!